conbench/app/_plots.py

```diff
@@ -249,6 +258,13 @@ def _source(
 
         dsdict["multisample_strings_with_unit"] = strings
 
+    if break_line_ixs:
+        # This source will be used for a line, and we want to "break" the line between
+        # the given indexes and the immediately previous data points. Do this by
+        # inserting np.NaNs.
```

Comment:
Did you find good docs explaining this behavior?

I at least found that it seems to be intentional :)
https://stackoverflow.com/a/67026207
https://github.com/PatrikHlobil/Pandas-Bokeh/issues/71

Reply:
Nope, I found it on StackOverflow as well, but I'll explore a little more!

Reply:
https://docs.bokeh.org/en/3.0.3/docs/user_guide/basic/lines.html#missing-points